User belongs to experiment group 'AlwaysDisplayTestExplorer - experiment'
User belongs to experiment group 'ShowPlayIcon - start'
User belongs to experiment group 'ShowExtensionSurveyPrompt - control'
User belongs to experiment group 'DebugAdapterFactory - experiment'
User belongs to experiment group 'PtvsdWheels37 - experiment'
User belongs to experiment group 'UseTerminalToGetActivatedEnvVars - control'
User belongs to experiment group 'AA_testing - experiment'
User belongs to experiment group 'LocalZMQKernel - experiment'
User belongs to experiment group 'CollectLSRequestTiming - control'
User belongs to experiment group 'CollectNodeLSRequestTiming - experiment'
User belongs to experiment group 'EnableIPyWidgets - experiment'
User belongs to experiment group 'RunByLine - experiment'
User belongs to experiment group 'CustomEditorSupport - control'
User belongs to experiment group 'pythonaa'
User belongs to experiment group 'pythonSendEntireLineToREPL'
User belongs to experiment group 'pythonInstallFlake8ButtonFirst'
> conda --version
> pyenv root
> python3.7 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> python3.6 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> python2 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> python ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> conda info --json
Error 2020-12-26 14:52:39: Failed to check if file needs to be fixed [EntryNotFound (FileSystemError): Unable to read file 'vscode-remote://ssh-remote+161.35.8.148/home/smarthome/.config/Code/User/settings.json' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open '/home/smarthome/.config/Code/User/settings.json')
	at _handleError (/home/smarthome/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:769:473)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async y.readText (/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/out/client/extension.js:9:262389)
	at async p.doesFileNeedToBeFixed (/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/out/client/extension.js:59:489303)
	at async /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/out/client/extension.js:59:488430
	at async Promise.all (index 0)
	at async p.getFilesToBeFixed (/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/out/client/extension.js:59:488376)
	at async p.updateTestSettings (/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/out/client/extension.js:59:488003)] {
  code: 'FileNotFound',
  name: 'EntryNotFound (FileSystemError)'
}
Starting Jedi Python language engine.
Python interpreter path: /usr/bin/python3
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.prefix)"
cwd: ~/smarthome
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.prefix)"
cwd: ~/smarthome
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
cwd: ~/smarthome
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
cwd: ~/smarthome
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"
cwd: ~/smarthome
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"
cwd: ~/smarthome
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py site --user-site
cwd: ~/smarthome
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py site --user-site
cwd: ~/smarthome
> conda --version
> pyenv root
> pyenv root
> pyenv root
> pyenv root
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py
Error 2020-12-26 14:52:43: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:584: DeprecationWarning: Deprecated since version 0.16.0. Use Script(...).get_names instead.
  jedi.api.names(

> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py pylint --disable=all --enable=F,unreachable,duplicate-key,unnecessary-semicolon,global-variable-not-assigned,unused-variable,unused-wildcard-import,binary-op-exception,bad-format-string,anomalous-backslash-in-string,bad-open-mode,E0001,E0011,E0012,E0100,E0101,E0102,E0103,E0104,E0105,E0107,E0108,E0110,E0111,E0112,E0113,E0114,E0115,E0116,E0117,E0118,E0202,E0203,E0211,E0213,E0236,E0237,E0238,E0239,E0240,E0241,E0301,E0302,E0303,E0401,E0402,E0601,E0602,E0603,E0604,E0611,E0632,E0633,E0701,E0702,E0703,E0704,E0710,E0711,E0712,E1003,E1101,E1102,E1111,E1120,E1121,E1123,E1124,E1125,E1126,E1127,E1128,E1129,E1130,E1131,E1132,E1133,E1134,E1135,E1136,E1137,E1138,E1139,E1200,E1201,E1205,E1206,E1300,E1301,E1302,E1303,E1304,E1305,E1306,E1310,E1700,E1701 --msg-template='{line},{column},{category},{symbol}:{msg}' --reports=n --output-format=text ~/smarthome/smarthomeproj/server/licensePlateRecognition.py
cwd: ~/smarthome
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/pyvsc-run-isolated.py pylint --disable=all --enable=F,unreachable,duplicate-key,unnecessary-semicolon,global-variable-not-assigned,unused-variable,unused-wildcard-import,binary-op-exception,bad-format-string,anomalous-backslash-in-string,bad-open-mode,E0001,E0011,E0012,E0100,E0101,E0102,E0103,E0104,E0105,E0107,E0108,E0110,E0111,E0112,E0113,E0114,E0115,E0116,E0117,E0118,E0202,E0203,E0211,E0213,E0236,E0237,E0238,E0239,E0240,E0241,E0301,E0302,E0303,E0401,E0402,E0601,E0602,E0603,E0604,E0611,E0632,E0633,E0701,E0702,E0703,E0704,E0710,E0711,E0712,E1003,E1101,E1102,E1111,E1120,E1121,E1123,E1124,E1125,E1126,E1127,E1128,E1129,E1130,E1131,E1132,E1133,E1134,E1135,E1136,E1137,E1138,E1139,E1200,E1201,E1205,E1206,E1300,E1301,E1302,E1303,E1304,E1305,E1306,E1310,E1700,E1701 --msg-template='{line},{column},{category},{symbol}:{msg}' --reports=n --output-format=text ~/smarthome/smarthomeproj/server/licensePlateRecognition.py
cwd: ~/smarthome
Error 2020-12-26 14:52:44: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the line is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the column is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Deprecated since version 0.17.0. Use the project API instead, which means Script(project=Project(dir, sys_path=sys_path)) instead.
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:626: DeprecationWarning: Deprecated since version 0.16.0. Use Script(...).infer instead.
  script.goto_definitions(), request["id"]

##########Linting Output - pylint##########
************* Module smarthomeproj.server.licensePlateRecognition
2,0,error,import-error:Unable to import 'imutils'
4,0,error,import-error:Unable to import 'pytesseract'
9,10,error,no-member:Module 'cv2' has no 'imread' member
9,57,error,no-member:Module 'cv2' has no 'IMREAD_COLOR' member
11,10,error,no-member:Module 'cv2' has no 'resize' member
13,11,error,no-member:Module 'cv2' has no 'cvtColor' member
13,29,error,no-member:Module 'cv2' has no 'COLOR_BGR2GRAY' member
14,11,error,no-member:Module 'cv2' has no 'bilateralFilter' member
16,12,error,no-member:Module 'cv2' has no 'Canny' member
17,15,error,no-member:Module 'cv2' has no 'findContours' member
17,46,error,no-member:Module 'cv2' has no 'RETR_TREE' member
17,61,error,no-member:Module 'cv2' has no 'CHAIN_APPROX_SIMPLE' member
19,38,error,no-member:Module 'cv2' has no 'contourArea' member
24,15,error,no-member:Module 'cv2' has no 'arcLength' member
25,17,error,no-member:Module 'cv2' has no 'approxPolyDP' member
38,8,error,no-member:Module 'cv2' has no 'drawContours' member
41,16,error,no-member:Module 'cv2' has no 'drawContours' member
42,16,error,no-member:Module 'cv2' has no 'bitwise_and' member
41,4,warning,unused-variable:Unused variable 'new_image'

----------------------------------------------------------------------
Your code has been rated at -13.33/10 (previous run: -13.33/10, +0.00)

Error 2020-12-26 14:54:10: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:584: DeprecationWarning: Deprecated since version 0.16.0. Use Script(...).get_names instead.
  jedi.api.names(

Error 2020-12-26 14:54:20: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the line is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the column is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Deprecated since version 0.17.0. Use the project API instead, which means Script(project=Project(dir, sys_path=sys_path)) instead.
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:626: DeprecationWarning: Deprecated since version 0.16.0. Use Script(...).infer instead.
  script.goto_definitions(), request["id"]

Error 2020-12-26 14:54:20: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the line is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the column is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Deprecated since version 0.17.0. Use the project API instead, which means Script(project=Project(dir, sys_path=sys_path)) instead.
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:626: DeprecationWarning: Deprecated since version 0.16.0. Use Script(...).infer instead.
  script.goto_definitions(), request["id"]

Error 2020-12-26 14:54:52: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the line is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(

Error 2020-12-26 14:54:52: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the column is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(

Error 2020-12-26 14:54:52: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Deprecated since version 0.17.0. Use the project API instead, which means Script(project=Project(dir, sys_path=sys_path)) instead.
  script = jedi.Script(
/home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:626: DeprecationWarning: Deprecated since version 0.16.0. Use Script(...).infer instead.
  script.goto_definitions(), request["id"]

Error 2020-12-26 14:54:52: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the line is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(

Error 2020-12-26 14:54:52: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Providing the column is now done in the functions themselves like `Script(...).complete(line, column)`
  script = jedi.Script(

Error 2020-12-26 14:54:52: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:592: DeprecationWarning: Deprecated since version 0.17.0. Use the project API instead, which means Script(project=Project(dir, sys_path=sys_path)) instead.
  script = jedi.Script(

Error 2020-12-26 14:54:52: stderr jediProxy Error (stderr) /home/smarthome/.vscode-server/extensions/ms-python.python-2020.12.424452561/pythonFiles/completion.py:626: DeprecationWarning: Deprecated since version 0.16.0. Use Script(...).infer instead.
  script.goto_definitions(), request["id"]

